requirements.txt

```diff
@@ -1,30 +1,30 @@
 # Base system
-Django==1.11.4
-psycopg2==2.7.3.1
+Django==1.11.7
+psycopg2==2.7.3.2
 Unipath==1.1
 dj-database-url==0.4.2
-djangorestframework==3.6.3
+djangorestframework==3.7.3
 django-dotenv==1.4.1
-Pillow==4.2.1
+Pillow==4.3.0
 
 # Useful libs
 django-messagegroups==0.4.5
 django-model-utils==3.0.0
 django-bootstrap-form==3.3
 sorl_thumbnail==12.3
-django-crispy-forms==1.6.1
+django-crispy-forms==1.7.0
 
 # Dev
-django-extensions==1.8.1
+django-extensions>=1.9,<2
```

Comment:
Why not use the [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) specifier `~=`?

Reply:
I didn't know about those, to be honest. But they seem to have a few gotchas, e.g. "This operator MUST NOT be used with a single segment version number such as ~=1.". I'm not sure if sticking with explicit ranges is better because it's clearer what it means.

Reply:
Of course it won't work with a single segment version number, what would be the point in it? It would be like not having a version specifier at all.